From 65e184365623f9baf21ec51ce95d13c033ff7684 Mon Sep 17 00:00:00 2001
From: momo-i <webmaster@momo-i.org>
Date: Sat, 11 Mar 2017 17:19:19 +0900
Subject: [PATCH 1/4] squid-4.0.18: config

---
 src/cf.data.pre | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/cf.data.pre b/src/cf.data.pre
index 56b3240..4c56604 100644
--- a/src/cf.data.pre
+++ b/src/cf.data.pre
@@ -4698,7 +4698,7 @@ DOC_END
 
 NAME: logfile_rotate
 TYPE: int
-DEFAULT: 10
+DEFAULT: 0
 LOC: Config.Log.rotateNumber
 DOC_START
 	Specifies the default number of logfile rotations to make when you
@@ -6534,11 +6534,11 @@ COMMENT_END
 
 NAME: cache_mgr
 TYPE: string
-DEFAULT: webmaster
+DEFAULT: root
 LOC: Config.adminEmail
 DOC_START
 	Email-address of local cache manager who will receive
-	mail if the cache dies.  The default is "webmaster".
+	mail if the cache dies.  The default is "root".
 DOC_END
 
 NAME: mail_from
-- 
2.9.3

